runtime/common/src/impls.rs

```diff
@@ -23,16 +20,40 @@ use sp_std::vec::Vec;
 
 common_types::impl_tranche_token!();
```

Comment:
I'm wondering why this sentence is a macro (since internally seems to not make use of any macro features). Could it be implemented in `common-types` directly without using `macro_rules!`, as plain Rust code, and later include it as `use common_types::TrancheToken` if needed?

Implementation: https://github.com/centrifuge/centrifuge-chain/blob/97f6ee328f5b0d1f2b08a010401672cbc80dea7b/libs/common-types/src/tokens.rs#L42

Reply:
https://github.com/centrifuge/centrifuge-chain/pull/807#issuecomment-1178485832